ui/assets/js/hound.js

```diff
@@ -693,7 +693,7 @@ var ContentFor = function(line, regexp) {
 
 var FileContentView = React.createClass({
   getInitialState: function() {
-    return { open: true };
+    return { open: !this.props.isAutoGenerated };
```

Comment:
It could be nice to put an `[AUTOGENERATED]` badge next to the file path. That's  not a blocker of course, but it would be a nice visual indication about why the file is collapsed.

Reply:
Great idea, will do!